HopsanGUI/ModelHandler.cpp

```diff
@@ -212,14 +213,14 @@ ContainerObject *ModelHandler::getCurrentViewContainerObject()
     return 0;
 }
 
-LogDataHandler2 *ModelHandler::getCurrentLogDataHandler()
+QSharedPointer<LogDataHandler2> ModelHandler::getCurrentLogDataHandler()
 {
     ModelWidget *pMW = getCurrentModel();
     if (pMW)
     {
         return pMW->getLogDataHandler();
     }
-    return 0;
+    return QSharedPointer<LogDataHandler2>(nullptr);
```

Comment:
Not that it makes any difference, but in this case returning nullptr directly should work since the type is known. But an alternative is to use the universal initializer (I think its called that) {}  ... (no need to change)

Reply:
"return 0" worked fine on my computer, but failed both on Travis and AppVeyor. I guess it's a Qt4 problem.